packages/block-library/src/cover-image/index.js

```diff
@@ -23,6 +23,7 @@ import {
 	withColors,
 	getColorClassName,
 } from '@wordpress/editor';
+import deprecated from '@wordpress/deprecated';
 
 const validAlignments = [ 'left', 'center', 'right', 'wide', 'full' ];
```

Comment:
Shouldn't this constant use CAPS_CASE like the constants in `edit.js`?

Reply:
Whoops, I just noticed this is the deprecated block. Well, I guess the point still applies, but it isn't really important. :stuck_out_tongue: